packages/server/src/lib/email.js

```diff
@@ -187,29 +190,37 @@ async function sendGrantAssignedEmail({ grantId, agencyIds, userId }) {
 }
 
 async function sendGrantDigestForAgency(agency) {
+    console.log(`${agency.name} is subscribed for notifications on ${moment().format('YYYY-MM-DD')}`);
     const newGrants = await db.getNewGrantsForAgency(agency);
 
     if (newGrants.length === 0) {
+        console.log(`${agency.name} has no new grants on ${moment().format('YYYY-MM-DD')}`);
+        return undefined;
+    }
+
+    const recipients = await db.getUsersByAgency(agency.id);
+    if (recipients.length === 0) {
+        console.log(`${agency.name} has no users for grants digest on ${moment().format('YYYY-MM-DD')}`);
         return undefined;
     }
 
     const grantDetails = [];
-    newGrants.slice(0, 3).forEach((grant) => grantDetails.push(module.exports.getGrantDetail(grant)));
+    newGrants.forEach((grant) => grantDetails.push(module.exports.getGrantDetail(grant)));
 
     const formattedBodyTemplate = fileSystem.readFileSync(path.join(__dirname, '../static/email_templates/_formatted_body.html'));
     const contentSpacerTemplate = fileSystem.readFileSync(path.join(__dirname, '../static/email_templates/_content_spacer.html'));
     const contentSpacerStr = contentSpacerTemplate.toString();
 
     let additionalBody = grantDetails.join(contentSpacerStr);
 
-    if (newGrants.length > 3) {
+    if (newGrants[0].total_grants > 3) {
         const additionalButtonTemplate = fileSystem.readFileSync(path.join(__dirname, '../static/email_templates/_additional_grants_button.html'));
         additionalBody += mustache.render(additionalButtonTemplate.toString(), { additional_grants_url: process.env.WEBSITE_DOMAIN });
     }
 
     const formattedBody = mustache.render(formattedBodyTemplate.toString(), {
         body_title: 'New grants have been posted',
-        body_detail: `There are ${newGrants.length} new grants matching your agency's keywords and settings.`,
+        body_detail: `There are ${newGrants[0].total_grants} new grants matching your agency's keywords and settings.`,
```

Comment:
IMO it's fine for now (I think people are generally used to this type of "default plural" behavior), but as we add more and more emails, it would be nice to have some kind of conventional/parameterized way of conditionally formatting plural/singular words so that we can avoid errant plurals in cases when the content is singular (e.g. "There is 1 new grant matching..." instead of "There are 1 new grants matching..."). 

Perhaps mustache has something akin to Django's [pluralize filter](https://docs.djangoproject.com/en/4.1/ref/templates/builtins/#pluralize)?